specification/data-resource-semantic-conventions.md

```diff
@@ -57,17 +57,24 @@ namespace = Company
 service.name = Shop.shoppingcart
 ```
 
-## Library
+## Telemetry SDK
 
-**type:** `library`
+**type:** `telemetry.sdk`
 
-**Description:** Telemetry library information.
+**Description:** The SDK used to produce telemetry data.
+
+In OpenTelemetry, this is the OpenTelemetry SDK implementation being used.  
+If the default SDK provided by the OpenTelemetry project is used, `telemetry.sdk.name` MUST
+be set to `opentelemetry-sdk`.  
+If another SDK, like a fork or a vendor-provided SDK is used, `telemetry.sdk.name` MUST be set
+to the fully-qualified class or module name of that SDK's main entry point, depending on the language.
+This value should be stable across different versions of an implementation.
 
 | Attribute  | Description  | Example  | Required? |
 |---|---|---|---|
-| library.name | The name of the telemetry library. | `opentelemetry` | No |
-| library.language | The language of telemetry library and of the code instrumented with it. <br/> The following spelling SHOULD be used for language strings: "cpp", "dotnet", "erlang", "go", "java", "nodejs", "php", "python", "ruby", "webjs" | `java` | No |
-| library.version | The version string of the library as defined in [Version Attributes](#version-attributes). | `semver:1.2.3` | No |
+| telemetry.sdk.name | The name of the telemetry SDK. | `opentelemetry-sdk` | No |
```

Comment:
What about making this required? This would greatly benefit supportability.

Reply:
It's appealing and I was tempting to do it but didn't want to change this in a clarification PR.

Reply:
Fair enough. Let's not forget about that, though.